OpenRA.Mods.Common/Widgets/Logic/Editor/ActorEditLogic.cs

```diff
@@ -309,6 +309,28 @@ public override void Tick()
 							slider.OnChange += value => so.OnChange(actor, value);
 							slider.OnChange += value => editorActionHandle.OnChange(value);
 
+							var valueField = sliderContainer.Get<TextFieldWidget>("VALUE");
```

Comment:
Does anyone have any thoughts on _requiring_ this text field on the actor props widget?

Reply:
I think all relevant yaml files have been modified. But yes, someone should double check that. It would be interesting to know if there are any other (valid) cases where the field would not exist.

Reply:
The field would not exist in 3rd-party games that haven't set it up (_may_ not want it, but I'm trying to determine that by asking for input), for the default games it is fine as you said.

Reply:
I changed the code to use GetOrNull